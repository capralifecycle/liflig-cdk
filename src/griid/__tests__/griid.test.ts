import { App, Stack } from "@aws-cdk/core"
import "jest-cdk-snapshot"
import { getGriidArtefactBucket } from "../"

test("getGriidArtefactBucket", () => {
  const app = new App()
  const stack = new Stack(app, "Stack", {
    env: {
      region: "eu-west-1",
    },
  })

  getGriidArtefactBucket(stack)
  expect(stack).toMatchCdkSnapshot()
})
