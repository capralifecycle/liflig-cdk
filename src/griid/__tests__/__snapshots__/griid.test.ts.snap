// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getGriidArtefactBucket 1`] = `Object {}`;
